extension/package.json

```diff
@@ -23,10 +23,9 @@
 	},
 	"devDependencies": {
 		"@types/firefox-webext-browser": "94.0.1",
-		"@types/node": "16.11.40",
+		"@types/node": "18.11.12",
```

Comment:
```suggestion
```
This project shouldn't have a dependency on this as it is a web targeted project.